.github/workflows/continuous-integration.yml

```
@@ -99,14 +105,21 @@ jobs:
             ORM_PROXY_IMPLEMENTATION: "${{ matrix.proxy }}"
 
       - name: "Upload coverage file"
-        uses: "actions/upload-artifact@v5"
+        uses: "actions/upload-artifact@v4"
         with:
           name: "phpunit-${{ matrix.extension }}-${{ matrix.php-version }}-${{ matrix.dbal-version }}-${{ matrix.proxy }}-coverage"
           path: "coverage*.xml"
 
 
   phpunit-postgres:
-    name: "PHPUnit with PostgreSQL"
+    name: > 
+      PostgreSQL - 
+      ${{ format('PHP {0} - DBAL {1} - PGSQL {2} - ext. {3}',
```

```suggestion
      ${{ format('PHP {0} - DBAL {1} - v. {2} - ext. {3}',
```

`PostgreSQL` is already shown before, we can just use `v.` for the version of PGSQL.

---

Maybe we can move the Postgres version to the front of the string? Something like…

> PostgreSQL 17 – PHP 8.4 – DBAL default – ext-pgsql

---

Thanks, that’s a simple but great idea!